src/rust/ide/lib/view/Cargo.toml

```diff
@@ -9,6 +9,7 @@ crate-type = ["cdylib", "rlib"]
 
 [dependencies]
 ast                  = { version = "0.1.0", path = "../ast/impl"                    }
+parser               = { version = "0.1.0", path = "../parser"                      }
```

Comment:
is this added only to make the debug scene working? I'm very against this change because it adds parser as a dependency to debug scenes, and thus, makes testing the GUI without backend very heavyweight. If you want to create a debug scene with a parser, please create separate crate for it. Debug scenes should be very lightweight - no parsers transpiled from scala, no engine, just rendering - this allows us benchmark this part.

Reply:
But documentation view uses parser, doesn't it? To convert the docstring to the html. So for me this dependency seems legit, and it's not only for debug scenes. And I don't see any way to avoid that and have visualizations working.